index.html

```diff
@@ -123,7 +125,72 @@ <h1> All Questions </h1>
+                                <span> No accepted answer </span>
+                            </label>
+                            <label>
+                                <input type="checkbox" value="has_bounty" name="filter_by">
+                                <span> Has bounty </span>
+                            </label>
+                        </div>
+                        <div class="flex column p-4 leading-7">
+                            <p class="font-semibold"> Sorted by </p>
+                            <label>
+                                <input type="radio" value="newest" name="sorted_by">
+                                <span> Newest </span>
+                            </label>
+                            <label>
+                                <input checked type="radio" value="recent_activity" name="sorted_by">
+                                <span> Recent activity </span>
+                            </label>
+                            <label>
+                                <input type="radio" value="highest_score" name="sorted_by">
+                                <span> Highest score </span>
+                            </label>
+                            <label>
+                                <input type="radio" value="most_frequent" name="sorted_by">
+                                <span> Most frequent </span>
+                            </label>
+                            <label>
+                                <input type="radio" value="bounty_ending_soon" name="sorted_by">
+                                <span> Bounty ending soon </span>
+                            </label>
+                        </div>
+                        <div class="flex column p-4 leading-7">
+                            <p class="font-semibold"> Tagged with </p>
+                            <label>
+                                <input checked type="radio" value="my_watched_tags" name="tagged_with">
+                                <span> My watched tags </span>
+                            </label>
+                            <label>
+                                <input type="radio" value="the_following)tags" name="tagged_with">
+                                <span> The following tags: </span>
+                                <div class="p-2"> <input class="text-input form-input-size"
+                                        placeholder="e.g javascript or phyton" type="text" name="search">
+                                    <input hidden type="submit" value="Search" />
+                                </div>
+                            </label>
+                        </div>
+                    </div>
+                    <hr>
+                    <div class="flex justify-between">
+                        <p class="btn btn-blue m-4 font-14 p-2"> Apply filter </p>
+                        <p class="btn-transparent font-14 p-2 m-4"> Cancel </p>
+                    </div>
```

Comment:
Wrong element. They are meant to be clicked and something will happen. Should be buttons.